ToDo.DataAccess/ToDo.ConsoleView/Program.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore;
+using Microsoft.Extensions.Configuration;
+using System;
+using ToDo.DataAccess.DataBase;
+using ToDo.DataAccess.Models;
+using ToDo.DataAccess;
+using Microsoft.EntityFrameworkCore;
+
+namespace ToDo.ConsoleView
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            CreateWebHostBuilder(args).Build().Run();
```

Comment:
We agreed to create an empty console application. Instead, you've created a web server. Please change it.

Reply:
I created, but didn't use it. Deleted.